tests/native-lib/fail/private_function.rs

```diff
@@ -0,0 +1,14 @@
+//@ignore-target-windows
```

Comment:
```suggestion
// Only works on Unix targets
//@ignore-target-windows
```
You missed that here, and in `tests/native-lib/fail/function_not_in_so.rs` as well.